packages/feedback/src/util/prepareFeedbackEvent.ts

```diff
@@ -0,0 +1,46 @@
+import type {Scope} from '@sentry/core';
+import {prepareEvent} from '@sentry/core';
+import type { Client } from '@sentry/types';
+
+import type { FeedbackEvent } from '../types';
+
+/**
+ * Prepare a feedback event & enrich it with the SDK metadata.
+ */
+export async function prepareFeedbackEvent({
+  client,
+  scope,
+  event,
+}: {
+  client: Client;
+  event: FeedbackEvent;
+  scope: Scope;
+}): Promise<FeedbackEvent | null> {
+  const preparedEvent = (await prepareEvent(
```

Comment:
before `prepareEvent`, we should also pass this through preprocess:

```suggestion
  const eventHint = { integrations: undefined };
  if (client.emit) {
    client.emit('preprocessEvent', event, eventHint);
  }
  
  const preparedEvent = (await prepareEvent(
```